pkg/microservice/aslan/core/workflow/handler/policy.yaml

```diff
@@ -88,3 +88,11 @@ rules:
         endpoint: "/api/directory/workflowTask/id/?*/pipelines/?*/cancel"
       - method: GET
         endpoint: "/api/aslan/delivery/releases"
+      - method: GET
+        endpoint: "/api/aslan/environment/environments"
+        resourceType: "Environment"
+        matchAttributes:
+          - key: "placeholder"
+            value: "placeholder"
```

Comment:
what does 「key = placeholder / value = placeholder」mean ?

Reply:
to make sure it will return an empty list instead of a 403 error